test/mtz-update-password_test.html

```diff
@@ -227,6 +228,60 @@
             expect(retype.validate).to.have.been.calledWith(element.retypeValue);
           });
         });
+        describe('strength meter', () => {
+          it('should call __updateStrengthMeter when password value changes', () => {
+            element.showStrengthMeter = true;
+            element.__updateStrengthMeter = sinon.spy();
+            expect(element.__updateStrengthMeter).not.to.have.been.called;
+            element.value = 'Pass@1234';
+            expect(element.__updateStrengthMeter).to.have.been.called;
+          });
+          it('should set strength properties when password value changes', () => {
+            element.showStrengthMeter = true;
+            element.value = 'Pass@1234';
+          });
+          it('should not set strength properties or lazy load zxcvbn when showStrengthMeter is false', () => {
+            element.showStrengthMeter = false;
+            element.value = 'Pass@1234';
+            element.importLazyGroup = sinon.stub();
+            expect(element.importLazyGroup).not.to.have.been.called;
+            expect(element.__passwordStrengthMeterValue).to.equal(undefined);
+            expect(element.__passwordStrengthTextValue).to.equal('');
+          });
+        });
+
+        describe('reveal visibility', () => {
+          it('should hide toggle-icon by default', () => {
+            const reveal = Polymer.dom(element.root).querySelector('#reveal');
+            expect(reveal).to.equal(null);
+          });
+        });
+
+        describe('reveal funcitonality', () => {
+          let reveal;
+          beforeEach((done) => {
+            element.reveal = true;
+            flush(() => {
+              reveal = Polymer.dom(element.root).querySelector('#reveal');
+              done();
+            });
+          });
+          it('should show toggle-icon when reveal is true', () => {
+            expect(reveal.checked).to.equal(false);
+          });
+          it('should toggle password type when toggle icon is checked', () => {
+            reveal.click();
+            element.__setPasswordType = sinon.spy();
+            flush(() => {
+              const password =Polymer.dom(element.root).querySelector('#password');
```

Comment:
This line should be failing lintine rules

Reply:
fixed